docs/GettingStartedDocs/Contributors/NonAccMachineSGXLinuxGettingStarted.md

```diff
@@ -72,21 +72,21 @@ sudo apt-get update
 
 - Or Install Intel DCAP Quote Provider with dpkg manually
 
-The [libsgx-dcap-default-qpl directory](https://download.01.org/intel-sgx/sgx-dcap/1.6/linux/distro/ubuntuServer18.04/debian_pkgs/libs/libsgx-dcap-default-qpl/) lists all different version of libsgx-dcap-default-qpl, please download the most recent version that matches your OS version. For Ubuntu 18.04 (code name [Bionic Beaver](https://wiki.ubuntu.com/BionicBeaver)), please download the version libsgx-dcap-default-qpl_{version}-bionic1_amd64.deb. For Ubuntu 16.04 (code name [Xenial Xerus](https://wiki.ubuntu.com/XenialXerus)), please download the version libsgx-dcap-default-qpl_{version}-xenial1_amd64.deb.
+The [libsgx-dcap-default-qpl directory](https://download.01.org/intel-sgx/sgx-dcap/1.8/linux/distro/ubuntu18.04-server/debian_pkgs/libs/libsgx-dcap-default-qpl/) lists all different version of libsgx-dcap-default-qpl, please download the most recent version that matches your OS version. For Ubuntu 18.04 (code name [Bionic Beaver](https://wiki.ubuntu.com/BionicBeaver)), please download the version libsgx-dcap-default-qpl_{version}-bionic1_amd64.deb. For Ubuntu 16.04 (code name [Xenial Xerus](https://wiki.ubuntu.com/XenialXerus)), please download the version libsgx-dcap-default-qpl_{version}-xenial1_amd64.deb.
```

Comment:
ubuntuServer18.04 -> ubuntu18.04-server seems to just be a rename

Reply:
Yeah, hashes of the files under each folders are the same. Clicking through the directories from the website leads me to believe that ubuntu<version>-server is the latest.